packages/support/resources/views/components/button/index.blade.php

```diff
@@ -183,7 +187,8 @@
         x-data="{}"
     @endif
     @if ($keyBindings)
-        x-mousetrap.global.{{ collect($keyBindings)->map(fn (string $keyBinding): string => str_replace('+', '-', $keyBinding))->implode('.') }}
+        id="{{ $keyBindingsAsString }}"
+        x-mousetrap.global.{{ $keyBindingsAsString }}="document.getElementById(@js($keyBindingsAsString)).click()"
```

Comment:
Can we not use `$el.click()` instead of setting an ID?

Reply:
That was my first idea, but it’s not working as expected. :(


https://github.com/filamentphp/filament/assets/4639175/5fe46759-e03d-4b0a-85d6-622794e16753

Reply:
Can we use this? I just really want to avoid ID conflicts

https://alpinejs.dev/magics/id

Reply:
Good idea! I will try to implement it.

Reply:
@danharrin Seems magic works :)


https://github.com/filamentphp/filament/assets/4639175/e0fe7b10-a853-4a0e-be4e-b8d2c769b443